### PR TITLE
Adds a new artifact type: Pitcher (+ a couple drinking glass string fixes)

### PR DESCRIPTION
**Pull request by frawhst**
## About the PR 
This adds a new type of artifact, similar to a pitcher. It has a large reagent limit, a random sip size, and comes with some funky
chems (somewhat) themed around the bar- what would strange alien lifeforms make a cocktail out of?
Created as a counterpart to the watering can, for the bar rather than the chem lab.

There's also two small string changes for drinking glasses:
1. When used inhand, glasses will now say "You can't think of anything to do with" and the object name, as opposed to
"... the glass". This prevents whatever artifact you're holding being called 'the glass', and means other glass types
are referred to more specifically ("you can't think of anything to do with the old-fashioned glass" etc etc.)
2. When interacting with a wedge on a drink rim, a duplicate "wedge" has been removed. Previously options would display as 
"[action] the [fruit] wedge wedge".

One slight concern on the code side is the use of a static list defined in New() to choose which filler drinks to add. From the code guidelines lists can be finicky, so feel free to bug me if that looks bad for performance.

## Why's this needed? 
Have you ever walked into the artlab and spied a beaker artifact? Have you even analyzed the contents to find a disgusting
mishmash of poisons, glowing chemicals, and explosives? Have you ever thought, "I want to chug this?"
*Now you can.*
It's also nice for the barkeep, since they can make use of the massive reagent limit, and allows you to easily imbibe large quantities of the drink of your choice (if you clean the gunk out first).
String changes are just touch-ups.

## Changelog

```
(u)Asche:
(+)Added a new artifact type.
```

### CODE REVIEW

**Comment by ThePotato97** [comment]
> beaker artifact


https://github.com/goonstation/goonstation/blob/67ea70925007edc32e0278c64abf6b46e9dbc380/code/obj/artifacts/artifact_items/watering_can.dm#L1-L4

**Comment by frawhst** [comment]
It's been suggested a bunch of times, and zjd said they'd add one on discord a month ago. I figure there's enough reason for it currently.

**Comment by ThePotato97** [comment]
> 
> 
> It's been suggested a bunch of times, and zjd said they'd add one on discord a month ago. I figure there's enough reason for it currently.

i wasn't saying this is a bad PR, just saying it's not a beaker

**Comment by frawhst** [comment]
Oh, my bad- misunderstood. True, but I always call it that because of the damn wiki page.

File: code/obj/artifacts/artifact_items/pitcher.dm [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+			reagents.add_reagent("voltagen", 50)
+		if (prob(5))
+			reagents.add_reagent("rainbow fluid", 30)
+		if (prob(1))
+			reagents.add_reagent("vampire_serum", 5)
+		if (prob(3))
+			reagents.add_reagent("painbow fluid", 10)
+		if (prob(1))
+			reagents.add_reagent("werewolf_serum", 2)
+		/* Removed pending patho rework, in case this becomes functional again
```
Comment (zjdtmkhzt) [inline code/obj/artifacts/artifact_items/pitcher.dm:64]: Liquid plasma is a disease reagent, doesn't really have anything to do with pathology. :)
But removing it is probably a good idea, seeing as the disease doesn't do anything.

**Comment by zjdtmkhzt** [comment]
Looks like a great patch, thanks for doing it so I don't have to actually do what I said I would.^^

I think this might have a very small merge conflict with #2215 because I renamed forceartitype to forceartiorigin to make room for a var that _actually_ forces the artifact type rather than the artifact origin, but it's nothing to worry about and should be easier to fix for whoever's patch gets merged second.

File: code/obj/artifacts/artifact_items/pitcher.dm (hunk 1 above)
Comment (frawhst) [reply code/obj/artifacts/artifact_items/pitcher.dm]: Ah, thanks for the clarification. Fully removed now.

**Review by Tarmunora — APPROVED** [review]

File: code/obj/artifacts/artifact_items/pitcher.dm [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+			reagents.add_reagent("madness_toxin", 10)
+		if (prob(20))
+			reagents.add_reagent("mutini", 50)
+		if(prob(3))
+			reagents.add_reagent("feather_fluid", 20)
+		if(prob(3))
+			reagents.add_reagent("bee", 10)
+		if(prob(15))
+			reagents.add_reagent("port", 30)
+		//Reagent IDs might be wrong for these two
```
Comment (ZeWaka) [inline code/obj/artifacts/artifact_items/pitcher.dm:120]:
```suggestion
```

**Review by ZeWaka — APPROVED** [review]

File: code/obj/artifacts/artifact_items/pitcher.dm [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	artifact = 1
+	module_research_no_diminish = 1
+	mat_changename = 0
+	can_recycle = 0
+
+	New(var/loc, var/forceartitype)
+		..()
+		var/datum/artifact/pitcher/AS = new /datum/artifact/pitcher(src)
+		if (forceartitype)
+			AS.validtypes = list("[forceartitype]")
```
Comment (zjdtmkhzt) [inline code/obj/artifacts/artifact_items/pitcher.dm:14]:
```suggestion
	New(var/loc, var/forceartiorigin)
		..()
		var/datum/artifact/pitcher/AS = new /datum/artifact/pitcher(src)
		if (forceartiorigin)
			AS.validtypes = list("[forceartiorigin]")
```
to make it also use the new name for the var that I changed in my patch that just got merged ^_^

Comment (frawhst) [reply code/obj/artifacts/artifact_items/pitcher.dm:14]: woops, didn't see that. Will add in a sec

**Pull request merged**